shared/ui/combobox/combobox.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { ChangeEvent, FC, MouseEventHandler, useState } from 'react';
+import { clsx } from 'clsx';
+
+import { Combobox as ComboboxUI } from '@headlessui/react';
+import * as ScrollArea from '@radix-ui/react-scroll-area';
+
+import style from './combobox.module.scss';
+
+import { Typography } from '..';
+
+import ArrowDown from '../../../assets/icons/arrow-down.svg?url';
+import Close from '../../../assets/icons/close.svg?url';
+
+import Image from 'next/image';
+
+type Option = {
```

Comment:
Is this type required for `ProfileForm`? A list of countries or cities can be (and probably will be) represented by a plain string array. How will you label the names of the cities? They already have full-fledged names, so labeling is pointless.

Reply:
Let's leave it for now, just in case we use it not only for a list of countries